core/src/replay_stage.rs

```diff
@@ -829,38 +827,6 @@ impl ReplayStage {
         (rv, Some((*best_bank).clone()))
     }
 
-    fn confirm_forks(
-        tower: &Tower,
-        stake_lockouts: &HashMap<u64, StakeLockout>,
-        total_staked: u64,
-        progress: &mut HashMap<u64, ForkProgress>,
-        bank_forks: &Arc<RwLock<BankForks>>,
-    ) {
-        progress.retain(|slot, prog| {
-            let duration = timing::timestamp() - prog.started_ms;
-            if tower.is_slot_confirmed(*slot, stake_lockouts, total_staked)
-                && bank_forks
-                    .read()
-                    .unwrap()
-                    .get(*slot)
-                    .map(|s| s.is_frozen())
-                    .unwrap_or(true)
-            {
-                info!("validator fork confirmed {} {}ms", *slot, duration);
-                datapoint_warn!("validator-confirmation", ("duration_ms", duration, i64));
```

Comment:
Does this break confirmation on metrics?

Reply:
oh true, i'll leave it without the filter then